tpp-prophets/protein_prophet.xml

```diff
@@ -25,11 +22,14 @@
             $unmapped
             $instances
             $delude
-          --minprob=$minprob
-          --minindep=$minindep
+          --minprob='$minprob'
+          --minindep='$minindep'
     </command>
     <inputs>
-        <param name="input_files" type="data" format="peptideprophet_pepxml,interprophet_pepxml" multiple="true" label="Peptide Prophet Results" help="These files will typically be outputs from peptide prophet or interprophet"/>
+        <param name="input_files" type="data" format="peptideprophet_pepxml,interprophet_pepxml" multiple="true"
+            label="Peptide Prophet Results" help="These files will typically be outputs from peptide prophet or interprophet"/>
+        <param argument="--database" type="data" format="fasta"
```

Comment:
Unfortunately, this tool does not seem to have a --database option and crashes because it can not find `/Users/icooke/Desktop/iptest/AASequences.fasta`

Reply:
Yep.  It's tricky to make this work ... but only needed for a test.  Can we find a workaround that only affects testing.  I would prefer to have left the main tool intact

Reply:
I think it's better if this option was left out.  It will most likely confuse users and is unlikely to work in most cases.